charts/nucliadb_reader/templates/reader.vs.yaml

```diff
@@ -52,6 +52,14 @@ spec:
             regex: '^/api/v\d+/kb/[^/]+/schema'
           method:
             regex: "GET|OPTIONS"
+        - uri:
+            regex: '^/api/v\d+/kb/[^/]+/search_configurations/[^/]+$'
+          method:
+            regex: "GET"
```

Comment:
options is needed too here I think

Reply:
It's being routed to the writer. I could add it here and it would be a conflict, although we don't care where it goes. In both cases, the response to options should be the same.